README.adoc

```diff
@@ -185,3 +185,21 @@ tuple ScalaModule =
   # We propogate errors
   global Error:              Option Error
 ----
+
+== Developer Documentation
+
+=== Running tests locally
+
+You must first have `wit` (v0.12.0), `wake` (v0.17.1), and `jq` (v1.5) installed.
+Note that only committed changes will be tested.
+
+----
+wit init test-ws -a $PWD -a git@github.com:sifive/environment-blockci-sifive.git::0.3.0
```

Comment:
Is this just for standalone testing? Such as:
```
$ git clone git@github.com:sifive/api-scala-sifive.git
$ cd api-scala-sifive
$ wit init test-ws -a $PWD -a git@github.com:sifive/environment-blockci-sifive.git::0.3.0
$ cd test-ws
$ ./api-scala-sifive/tests/run-tests.sh
```
Is it intended that workspace-in-a-workspace would happen too? Example:
```
$ wit init workspace -a git@github.com:sifive/product-coreip-sifive.git
$ cd workspace/api-scala-sifive
$ wit init test-ws -a $PWD -a git@github.com:sifive/environment-blockci-sifive.git::0.3.0
$ cd test-ws
$ ./api-scala-sifive/tests/run-tests.sh
```
Mostly just wondering if someone does the ws-in-ws and they find themselves with duplicate wake files if they go back to the exterior ws